### DISCUSSION TITLE
Add arg to docker build

### DISCUSSION THREAD

**Discussion by hamedyosefian** [coollabsio/coolify#1849] [Q&A]
How to use arg when run docker build similar this code 
docker build --build-arg SCOPE=dashboard

**Comment by OnDroo** [comment]
Hi,

Not sure if you already figured it out but for anyone looking for the same thing.
You can set it in the Project configuration under Environment Variables as a Build Variable.
![image](https://github.com/user-attachments/assets/22b9ea73-a285-4a01-ba8d-b5f2806c38ee)
